crates/turborepo-lib/src/config/mod.rs

```diff
@@ -37,6 +37,11 @@ pub enum Error {
          one"
     )]
     NoTurboJSON,
+    #[error(
+        "loginUrl is configured to \"{value}\", but cannot be a base URL. This happens in \
+         situations like using a `data:` URL."
+    )]
+    LoginUrlCannotBeABase { value: String },
```

Comment:
Upstream throws an empty-tuple error here, so we take and turn that into something more-useful.